src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
```

Comment:
I'm confused at how a circular reference could get introduced here.  Are we thinking the gssapi headers could start including our headers?